maqs-utilities/src/main/java/com/cognizantsoftvision/maqs/utilities/logging/IFileLogger.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 (C) Cognizant SoftVision, All rights Reserved
+ */
+
+package com.cognizantsoftvision.maqs.utilities.logging;
+
+/**
+ * The File Logger interface class.
+ */
+public interface IFileLogger extends ILogger {
+
+  /**
+   * Gets the FilePath value.
+   *
+   * @return returns the file path
+   */
+  String getFilePath();
+
+  /**
+   * Gets the Message Type value.
+   *
+   * @return The Message Type.
+   */
+  MessageType getMessageType();
```

Comment:
Why is this here and how is different from "getLoggingLevel" in the ILogger interface

Reply:
deleted this and changed getLoggingLevel to getMessageType